src/arguments.c

```diff
@@ -52,6 +52,10 @@ void cp_parse_cmd_args(struct cp_cmd_args * args, int argc, char** argv) {
 
             args->general_flags |= VERSION_FLAG;
 
+		} else if (argc == 2) {
+
+			args->filename = argv[1];
```

Comment:
I think `else { args->filename = argv[arg]; }` would work better. Also, the indentation is a bit messed up.